keylime/migrations/versions/8c0f8ded1f90_convert_allowlists_to_ima_policies.py

```diff
@@ -0,0 +1,140 @@
+"""convert allowlists to IMA policies
+
+Revision ID: 8c0f8ded1f90
+Revises: 039322ea079b
+Create Date: 2022-10-27 18:18:31.674283
+
+"""
+import copy
+import datetime
+import json
+
+import sqlalchemy as sa
+from alembic import op
+
+from keylime import keylime_logging
+from keylime.ima import ima
+
+logger = keylime_logging.init_logging("db_migrations")
+
+
+# revision identifiers, used by Alembic.
+revision = "8c0f8ded1f90"
+down_revision = "039322ea079b"
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()[f"upgrade_{engine_name}"]()
+
+
+def downgrade(engine_name):
+    globals()[f"downgrade_{engine_name}"]()
+
+
+def upgrade_registrar():
+    pass
+
+
+def downgrade_registrar():
+    pass
+
+
+def upgrade_cloud_verifier():
```

Comment:
So the upgrade is changing the values in `ima_policy` but still leaving the old names for the tables and fields? Could we take this opportunity to upgrade those as well? If we're going to pull off the band-aid, let's get it all off.

Reply:
As discussed in the Keylime video meeting today, let's hold off on database changes for a separate PR.